overlays/keystone/board/sifive/hifive-unmatched/sifive_unmatched_keystone_defconfig

```diff
@@ -0,0 +1,67 @@
+CONFIG_RISCV=y
```

Comment:
Potentially add a `uboot` prefix to the name of this file to indicate this is for configuring uboot.